krux_boto/__init__.py

```diff
@@ -175,7 +190,7 @@ def add_cli_arguments(self, parser):
 
 def main():
     app    = Application()
-    region = boto.ec2.get_region(app.boto.cli_region)
```

Comment:
it's a preference thing, but i'd rather remove the explicit import of 'get_region' and use the full call to boto.ec2.get_region - that way it's always easy for a future maintainer to see where the call is coming from, without having to jump through the file.

Totally a personal nit though.